### ISSUE TITLE
Add Fluxnet15 datasets to benchmark analysis 

### ISSUE THREAD

**Issue by davidjpmoore** [PalEON-Project/MIP-Change-and-Stability#3]
Add Fluxnet15 datasets to benchmark analysis for Harvard, Willow Creek & Sylvania

**Comment by crollinson** [comment]
@davidjpmoore file AnnualGPP_MIPBenchmark.FLUXNET.22Jun2016.RData has data for Morgan Monroe (not one of PalEON sites) and nothing for Howland.  Is it possible to early get Howard into these scripts & Rdata file?

**Comment by crollinson** [comment]
I guess I'm also a bit confused on the file structures.  Is it safe to aggregate the L2 data (issue #1?) to annual level or should I just rely on Fluxnet15 for the GPP values?

**Comment by davidjpmoore** [comment]
Yes - we are all confused by the structures - I'll speak to Dario this week - he'll be a fluxcourse

No it is not possible - I think that FLUXNET15 does not contain Howland - AMERIFLUX L2 contains Howland
